charts/enterprise-metrics/templates/alertmanager-dep.yaml

```diff
@@ -88,6 +88,8 @@ spec:
             - name: storage
               mountPath: "/data"
               subPath: {{ .Values.alertmanager.persistence.subPath }}
+            - name: temp
```

Comment:
nit: I slightly prefer the name `tmp` over `temp` but it's so nitty, feel free to ignore.

Reply:
done